### ISSUE TITLE
Remove references to pod-security-policies

### ISSUE THREAD

**Issue by cbron** [rancher/dashboard#9725]
**Is your feature request related to a problem? Please describe.**
Rancher 2.8.0 is dropping support for <1.25 clusters. Therefore no cluster that supports PSP's will be supported by Rancher. We can remove any references towards it. 

**Describe the solution you'd like**

The UI should remove any code around PSPs, and references to following endpoints:

* `/v3/podsecuritypolicytemplates`
* `/v3/podsecuritypolicytemplateprojectbindings`
* `/v1/management.cattle.io.podsecuritypolicytemplates`
* `/v1/management.cattle.io.podsecuritypolicytemplateprojectbindings`

Or references directly towards CRDs of: 
* upstream's PodSecurityPolicies
* podsecuritypolicytemplates
* podsecuritypolicytemplateprojectbindings

**Additional context**

Backend issue doing the same, will be blocked by this issue: https://github.com/rancher/rancher/issues/39789

**Comment by gaktive** [comment]
This is mostly for Vue. For Ember, RKE1 would need to be amended. We won't worry about the public cloud pages since we'll move over them over to Vue in due time.

**Comment by gaktive** [comment]
RKE1 could be split into its own ticket for Ember focus.

**Comment by eva-vashkevich** [comment]
Hi, @gaktive @kwwii should we add any banners notifying customers that support for 1.25 has been dropped as part of this ticket?

**Comment by cnotv** [comment]
That's a lot of work wasted.

**Comment by eva-vashkevich** [comment]
Hi @samjustus sorry I missed the update. yes, we removed all references to PSPs from the UI.
